src/server/src/batch.cpp

```diff
@@ -126,6 +126,7 @@ process_batch(stinger_t * S, StingerBatch & batch)
   StingerServerState & server_state = StingerServerState::get_server_state();
   min_batch_ts = std::numeric_limits<int64_t>::max();
   max_batch_ts = std::numeric_limits<int64_t>::min();
+  int64_t chunksize = 8192;
```

Comment:
It's a tradeoff between load balance and overhead. Each chunk dispatch requires the thread to synchronize with a centralized work queue. The scheduler will still break things into smaller chunks when there is less than a full chunk remaining, I believe. But testing is the best way to know for sure.